ntoskrnl/io/pnpmgr/plugplay.c

```diff
@@ -245,8 +245,8 @@ IopInitializeDevice(
     }
 
     /* Set the device instance of the device node */
-    RtlDuplicateUnicodeString(0, &DeviceInstance, &DeviceNode->InstancePath);
-
+    StatusUnused = RtlDuplicateUnicodeString(0, &DeviceInstance, &DeviceNode->InstancePath);
```

Comment:
Maybe this also works:
`(VOID)RtlDuplicateUnicodeString(0, &DeviceInstance, &DeviceNode->InstancePath);`

Reply:
According to my local test, it didn't fix.